src/lib.rs

```diff
@@ -1,14 +1,230 @@
-pub fn add(left: usize, right: usize) -> usize {
-    left + right
+use ads1x1x::{
+    channel,
+    ic::{Ads1115, Resolution16Bit},
+    interface::I2cInterface,
+    Ads1x1x, SlaveAddr,
+};
+use ak09915_rs::{Ak09915, Mode};
+use bmp280::{Bmp280, Bmp280Builder};
+use embedded_hal::adc::OneShot; //used on ads
+use icm20689::{self, AccelRange, Builder, GyroRange, SpiInterface, ICM20689};
+use linux_embedded_hal::spidev::{self, SpidevOptions};
+use linux_embedded_hal::sysfs_gpio::Direction;
+use linux_embedded_hal::I2cdev;
+use linux_embedded_hal::{Delay, Pin, Spidev};
+use nb::block; //used on ads
+use pwm_pca9685::{Address, Channel, Pca9685};
```

Comment:
Address is error prone, I would recommend to access pwm_pca9685 directly than importing these generic names.

Reply:
could we keep as pwm_Address, adc_Address... and so on?